ResidentContactApi/V1/Domain/ResidentContactDetailsRecordToInvalidateNotFound.cs

```diff
@@ -0,0 +1,11 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace ResidentContactApi.V1.Domain
+{
+    public class ResidentContactDetailsRecordToInvalidateNotFoundException : Exception
```

Comment:
Wonder if we could make the name shorter?

```suggestion
    public class ContactRecordNotFoundException : Exception
```

Reply:
Does a Not found exception exist already? We might be able to use that rather than creating a new one?

Reply:
There is a not found exception however it is for resident which would make sense to add it here, I will reduce the name and then you can advise on what you think